### ISSUE TITLE
AI Shield Management in TvT Multiplayer questionable

### ISSUE THREAD

**Issue by z64555** [scp-fs2open/fs2open.github.com#360]
Stumbled on this while going through nixprime's shield management. Currently, AI have a difficulty factor applied to how frequently they manage their shields. This factor is only applied if the Player's team is hostile to the attacked ship's team.

function in question is ai_manage_shield() in aicode.cpp:12329.

My concern is how ai_manage_shield behaves in a Team vs. Team multiplayer match, whether all teams (with AI opponents) use the same shield management frequency or not.

**Comment by JohnAFernandez** [comment]
Yeah, it really looks like there's no way that it balances during TvT, let me start looking at ways to correct this.
